### PR TITLE
Update faqs.md

### PR DESCRIPTION
**Pull request by scjane**

### CODE REVIEW

File: docker-for-azure/faqs.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ $ sudo ping 10.0.0.4
 
 
 ## What are the different Azure Regions?
-All regions can be found here: [Microsoft Azure Regions](https://azure.microsoft.com/en-us/regions/)
+All regions can be found here: [Microsoft Azure Regions](https://azure.microsoft.com/en-us/regions/).
```
Comment (mstanleyjones) [inline docker-for-azure/faqs.md]: This should be a period rather than a comma.

Comment (scjane) [inline docker-for-azure/faqs.md]: Hi, do you mean the punctuation I added? It is a period (.), not a comma (,)😵

Comment (mstanleyjones) [inline docker-for-azure/faqs.md]: Oh wow, you are right. Sometimes it's hard to read details in fonts! Sorry about that.

Comment (scjane) [inline docker-for-azure/faqs.md]: Yep. Sometimes it's also hard for me to check what changes I have made.

**Pull request merged**